### DISCUSSION TITLE
Installation Frepple community edition 7.3.0 under linux mint localhost

### DISCUSSION THREAD

**Discussion by Telemark74** [frePPLe/frepple#584] [Q&A]
h Guru of Frepple,

is it possible to install Frepple community edition 7.3.0 under xampp ?
(there is not postgres database in xampp)

i tried in  following the documentation of Frepple 7.3.0 to install with blocked issue regarding the creation of the Frepple database postgres , is somebody have the linux script from scratch to install the Trepple under linux mint localhost ?

Regards.

**Comment by jdetaeye** [comment]
Frepple runs on ubuntu linux with a postgres database.   
You can use docker to run on any other operating system, but you'll still need a postgres database as the backend database.
